src/Functions/formatRow.cpp

```diff
@@ -40,8 +39,7 @@ class FunctionFormatRow : public IFunction
         , arguments_column_names(std::move(arguments_column_names_))
         , context(std::move(context_))
     {
-        if (!FormatFactory::instance().getAllFormats().contains(format_name))
```

Comment:
This is not the only place in the code where we use `FormatFactory::instance().getAllFormats().contains()`, we should check for all such usages and change them to smth like `FormatFactory::instance().has(format_name)`.

Reply:
Or we can use `KnownFormatNames::instance().exists(format_name)`